.github/workflows/publish-preview.yml

```diff
@@ -0,0 +1,46 @@
+name: Publish Preview
+
+# added new action for checking secrets to weed out forks (name tbd)
+
+on:
+  pull_request:
+    branches:
+      - master
+      - release-*
+
+jobs:
+  filter:
+    name: Fork Filter
+    runs-on: ubuntu-latest
+    steps:
+    # stage for adding in thefrontside/actions
+    - name: forking filter
+      uses: taras/georgia/.github/actions/filter-for-fork@release-x
+      env:
+        NPM_AUTH_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}
+
+  publish: 
+    name: Publish Commit Package
+    runs-on: ubuntu-latest
+    needs: filter
+    steps:
+    - uses: actions/checkout@v1
+    - name: Write NPM Snapshot Version
+      uses: thefrontside/actions/write-npm-snapshot-version@master
+    - name: NPM Publish Commit
+      uses: actions/npm@master
+      with: 
+        args: publish --access=public
+      env:
+        NPM_AUTH_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}
+
+  comment: 
+    name: Post Instruction Comment
+    runs-on: ubuntu-latest
+    needs: publish
+    steps:
+    - uses: actions/checkout@v1
+    - name: Post Comment
+      uses: thefrontside/actions/post-instruction-comment@master
```

Comment:
This action should be named "post npm usage instructions comment" or the like because as-is it feels like you could pass any block to it, but it is npm specific.

Reply:
updated